docs/sources/get-started/overview.md

```diff
@@ -1,76 +1,52 @@
+Loki is designed to be very cost effective and highly scalable. Unlike other logging systems, Loki does not index the contents of the logs, but only indexes metadata about your logs as a set of labels for each log stream. 
 
-## Loki features
+A log stream is a set of logs which share the same labels. Labels help Loki to find a log stream within your data store, so having a quality set of labels is key to efficient query execution.
 
--  **Efficient memory usage for indexing the logs**
+Log data is then compressed and stored in chunks in an object store such as Amazon S3 or Google Cloud Storage, or even, for development or proof of concept, on the filesystem. A small index and highly compressed chunks simplifies the operation and significantly lowers the cost of Loki.
 
-    By indexing on a set of labels, the index can be significantly smaller
-    than other log aggregation products.
-    Less memory makes it less expensive to operate.
+{{< figure  src="../loki-overview-2.png" caption="**Loki logging stack**" >}}
 
--  **Multi-tenancy**
+A typical Loki-based logging stack consists of 3 components:
 
-    Loki allows multiple tenants to utilize a single Loki instance.
-    The data of distinct tenants is completely isolated from other tenants.
-    Multi-tenancy is configured by assigning a tenant ID in the agent.
+- **Agent** - An agent or client, for example `promtail`, which is distributed with Loki, or the Grafana Agent. The agent scrapes logs, turns the logs into streams by adding labels, and pushes the streams to Loki through an HTTP API.
 
--  **LogQL, Loki's query language**
+- **Loki** - The main server, responsible for ingesting and storing logs and processing queries. It can be deployed in three different configurations, for more information see [deployment modes]{{< relref "../get-started/deployment-modes/" >}}.
+  
+- **[Grafana](https://github.com/grafana/grafana)** for querying and displaying log data. You can also query logs from the command line, using [LogCLI]({{< relref "../query/logcli" >}}) or using the Loki API directly.
 
-    Users of the Prometheus query language, PromQL, will find LogQL familiar
-    and flexible for generating queries against the logs.
-    The language also facilitates the generation of metrics from log data,
-    a powerful feature that goes well beyond log aggregation.
+## Loki features
 
--  **Scalability**
+- **Scalability** - Loki is designed for scalability, and can scale from as small as running on a Raspberry Pi to ingesting petabytes a day. 
+In its most common deployment, “simple scalable mode”, Loki decouples requests into separate read and write paths, so that you can independently scale them, which leads to flexible large-scale installations that can quickly adapt to meet your workload at any given time.
+If needed, each of Loki's components can also be run as microservices designed to run statelessly and natively within Kubernetes.
```

Comment:
`statelessly` - should we remove this word? atleast in the context of kubernetes we are not exactly stateless as some of the components expect state to be retained across restarts to ensure durability